scss/standalone/patterns_forms-combined.scss

```diff
@@ -0,0 +1,29 @@
+@import '../vanilla';
```

Comment:
Why do we need a separate CSS for combined example?

Reply:
@bartaz  Revised per MM discussion: https://chat.canonical.com/canonical/pl/jgkzwmoeejdgjr1zyjyax8zmhr